roles/linux_preparation/tasks/recore.yml

```diff
@@ -132,35 +75,6 @@
     - /etc/issue.net
     - /etc/issue
 
-- name: enable ssh
```

Comment:
This needs to stay enabled.

Reply:
Doesn't hurt to leave that in, as it will not break anything to check it is enabled.

Reply:
Actually, it was only moved. It is still present in the file.